### PR TITLE
chore: migrate to schema 71 (PB v0.16.0).

### PR DESCRIPTION
**Pull request by yajuaryarudderstack**
# Description

Migrate to schema 71

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Developer Checklist

- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] Lint rules pass locally
- [ ] If applicable, are your project model changes backward compatible. Note that inputs should not be removed or renamed as it will break existing projects.

## Code Reviewer Checklist

- [ ] This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] Migration to the latest schema, if required
- [ ] If applicable, are your project model changes backward compatible. Note that inputs should not be removed or renamed as it will break existing projects.

## Security

- [ ] The code changed/added as part of this pull request won't create any security issues with how the software is being used.

### CODE REVIEW

File: models/localMacros.yaml [hunk 1]
```diff
@@ -18,14 +18,4 @@ macros:
...
-        timestamp_diff(timestamp({{end_date_column}}), timestamp({{start_date_column}}), {{granularity}}) 
-      {% else %}
-        date_diff(date({{end_date_column}}), date({{start_date_column}}), {{granularity}}) 
-      {% endif %}
-    {% else %} 
-      datediff({{granularity}}, {{start_date_column}}, {{end_date_column}})
-    {% endif %}
-    "
\ No newline at end of file
+    value: " {% if warehouse.DatabaseType() == \"bigquery\" %} {% if granularity == 'second' %} timestamp_diff(timestamp({{end_date_column}}), timestamp({{start_date_column}}), {{granularity}}) {% else %} date_diff(date({{end_date_column}}), date({{start_date_column}}), {{granularity}}) {% endif %} {% else %} datediff({{granularity}}, {{start_date_column}}, {{end_date_column}}) {% endif %} "
```
Comment (tipped73) [inline models/localMacros.yaml]: should we revert this?

**Review by tipped73 — APPROVED** [review]
LGTM except for one comment

**Comment by yajuaryarudderstack** [comment]
Updated, please check @tipped73 @sprksh .

File: models/localMacros.yaml [hunk 2]
```diff
@@ -28,4 +28,4 @@ macros:
     {% else %} 
       datediff({{granularity}}, {{start_date_column}}, {{end_date_column}})
     {% endif %}
-    "
\ No newline at end of file
+
```
Comment (sprksh) [inline models/localMacros.yaml:31]: This is an issue

**Pull request merged**